### DISCUSSION TITLE
Are custom user profiles possible?

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#2608] [Q&A]
Is it possible to create unique configuration files/profiles (DNSCrypt-Proxy.toml, Cloaking Rules, etc) for specific users and keep those profiles in AppData Local or Roaming directories while keeping executable image somewhere else (like in Program Files directory) ?

**Comment by lifenjoiner** [comment]
Yes, and then you should specify the configuration file (dnscrypt-proxy.toml) with absolute path and start the App with `-config`.

**Comment by ghost** [comment]
What if I need to install DNSCrypt-Proxy as service, but have it loadi configuration TOML file from AppData directory?

**Reply by lifenjoiner** [reply]
Also specify `-config` for the installation command.

**Comment by ghost** [comment]
It works now. Many thanks!
